README.md

```diff
@@ -1,76 +1,50 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# Todo List
 
-> One paragraph statement about the project.
+> This is a minimal yet powerful to-do list.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./src/assets/app_screenshot.png)
 
-Additional description about the project and its features.
+The application has functionality to add, remove, mark as completed, and reorder todo item.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
-
-## Live Demo
-
-[Live Demo Link](https://livedemo.com)
-
+- HTML
+- CSS
+- Javascript
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
 ### Setup
 
-### Install
-
-### Usage
+1.  Open a terminal, navigate to the directory that you would like to clone the repository into using commands like:
 
-### Run tests
+    - `cd /pathname` to change your working directory.
 
-### Deployment
+2.  Next, clone the remote repository and create a local copy on your machine using this command:
 
+    - `git clone https://github.com/afizsavage/todo-list`
 
+3.  Now you can see the contents of the remote repository on your machine by moving into the newly cloned directory:
 
-## Authors
+    - `todo-list`
+    - `ls -la`
```

Comment:
- [ ] I would recommend writing complete steps. Here you have just written `Open the index.html file in the browser` but there are two folders, `dist` and `src` mention which one. Also, I would recommend writing the steps on installing npm package and running server using webpack like `npm i` and `npm start` since this project used webpack so it's better to give these steps.